src/tools/devtools/overview.md

```diff
@@ -37,10 +37,10 @@ your existing IDE or command-line based development workflow.
 
 <a id="install-devtools"></a>
 
-## How do I install DevTools?
+## How do I install and start DevTools?
 
 See the [VS Code][], [Android Studio/IntelliJ][], or
-[command line][] pages for installation instructions.
+[command line][] pages for instructions on how to install and start DevTools.
```

Comment:
I recommend we remove the word "install". This may have made sense when users had to run `pub global activate devtools` to get access to DevTools, but I don't think this makes sense anymore. The user doesn't have to "install" devtools - they get it automatically with their Flutter SDK.

So how about:

"How do I start DevTools?" and "...how to start DevTools"